internal/core/graphics/image.rs

```diff
@@ -1324,6 +1324,51 @@ pub(crate) mod ffi {
     ) {
         image.set_nine_slice_edges(top, right, bottom, left);
     }
+
+    #[no_mangle]
+    pub extern "C" fn slint_image_to_rgb8(
+        image: &Image,
+        data: &mut SharedVector<Rgb8Pixel>,
+        width: &mut u32,
+        height: &mut u32,
+    ) -> bool {
+        image.to_rgb8().map_or(false, |pixel_buffer| {
```

Comment:
this is `is_some_and()`
But it doesn't have to be changed now.  @hunger will change it later when he applies clippy suggestion :-)

Reply:
Hehe ok :)